### PR TITLE
How to update/pull

### PR DESCRIPTION
**Pull request by seyyah**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,15 @@ LittlevGL ported to STM32F746G-DISCO
...
 git clone --recursive https://github.com/littlevgl/stm32f746_disco_no_os_atollic.git
+```
 
 3. Import the project into your workspace.
 4. Build and run!
+
+# How to update/pull repo
+
+```sh
+git submodule update --init
```
Comment (embeddedt) [inline README.md:18]: @seyyah Don't we need `git pull` here as well?

Comment (kisvegabor) [reply README.md:18]: I also use `git pull` to update submodules. `git submodule update` doesn't work for me once the submodule is already there.

Comment (embeddedt) [reply README.md:18]: @seyyah See https://stackoverflow.com/a/5828396.

Comment (seyyah) [reply README.md:18]: You may be right. I'll try ASAP.

Comment (seyyah) [reply README.md:18]: You're right @kisvegabor @embeddedt

**Pull request closed without merging**